tooling/redhat-camel-spring-boot-bom/pom.xml

```diff
@@ -32,14 +32,16 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-	<!-- Camel / SpringBoot dependency versions -->
+        <!-- Camel / SpringBoot dependency versions -->
         <version.camel>3.20.1.redhat-00005</version.camel>
         <version.camel-spring-boot>3.20.1.redhat-00012</version.camel-spring-boot>
         <version.spring-boot>2.7.4</version.spring-boot>
 
         <!-- Third party dependency versions -->
         <version.snakeyaml>1.33</version.snakeyaml>
         <version.hsqldb>2.7.1</version.hsqldb>
+        <version.elasticsearch-java-client>8.5.2</version.elasticsearch-client>
```

Comment:
please fix typo `<version.elasticsearch-java-client>8.5.2</version.elasticsearch-client>` to `<version.elasticsearch-java-client>8.5.2</version.elasticsearch-java-client>` (the close tag)